api/src/queries/survey/survey-view-queries.ts

```diff
@@ -3,6 +3,45 @@ import { getLogger } from '../../utils/logger';
 
 const defaultLog = getLogger('queries/survey/survey-view-queries');
 
+/**
+ * SQL query to get all permit numbers applicable for a survey
+ *
+ * @param {number} projectId
+ * @returns {SQLStatement} sql query object
+ */
+export const getSurveyPermitNumbersSQL = (projectId: number): SQLStatement | null => {
```

Comment:
Similar to my previous comments (https://github.com/bcgov/biohubbc/pull/405/files#r668281986) we could name this something more verbose: "getAllAssignablePermitsForASurveySQL" or something like that.